lib/modules/platform/gerrit/scm.ts

```diff
@@ -34,7 +34,7 @@ export class GerritScm extends DefaultGitScm {
       .findChanges(repository, searchConfig, true)
       .then((res) => res.pop());
     if (change) {
-      return change.current_revision! as LongCommitSha;
+      return change.current_revision as LongCommitSha;
```

Comment:
unintentional?

Reply:
This diff should have disappeared once merged from main

Reply:
https://github.com/renovatebot/renovate/pull/28833